azure_pipelines/verify_msalcpp_per_pr_ios.yml

```diff
@@ -42,6 +42,13 @@ jobs:
      echo "##vso[task.setvariable variable=commit_number]$(git rev-parse HEAD)"
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->